clients/ops/admin-ui/src/features/datastore-connections/add-connection/manual/DSRCustomizationForm.tsx

```diff
@@ -0,0 +1,163 @@
+                      mb="6px"
+                      spacing="24px"
+                    >
+                      <Box w="416px">PII Field</Box>
+                      <Box w="416px">DSR Package Label</Box>
+                      <Box />
+                    </HStack>
+                    <Box
+                      maxH="calc(100vh - 484px)"
+                      paddingRight="13px"
+                      overflow="auto"
+                    >
+                      {fields && fields.length > 0
+                        ? fields.map((_field: Field, index: number) => (
+                            <HStack
+                              // eslint-disable-next-line react/no-array-index-key
+                              key={index}
+                              mt={index > 0 ? "12px" : undefined}
+                              spacing="24px"
+                            >
+                              <Box h="57px" w="416px">
+                                <CustomInput
+                                  autoFocus={index === 0}
+                                  displayHelpIcon={false}
+                                  isRequired
+                                  name={`fields.${index}.pii_field`}
+                                />
+                              </Box>
+                              <Box h="57px" w="416px">
+                                <CustomInput
+                                  displayHelpIcon={false}
+                                  isRequired
+                                  name={`fields.${index}.dsr_package_label`}
+                                />
+                              </Box>
+                              <Box
+                                h="57px"
+                                visibility={index > 0 ? "visible" : "hidden"}
+                              >
+                                <TrashCanSolidIcon
+                                  onClick={() => fieldArrayProps.remove(index)}
+                                  _hover={{ cursor: "pointer" }}
+                                />
+                              </Box>
+                            </HStack>
+                          ))
+                        : null}
+                    </Box>
+                    <Text
+                      color="complimentary.500"
+                      fontWeight="medium"
+                      fontSize="sm"
+                      mb="24px !important"
+                      mt="24px !important"
+                      onClick={() => {
+                        fieldArrayProps.push({
+                          pii_field: "",
+                          dsr_package_label: "",
+                        });
+                      }}
+                      _hover={{ cursor: "pointer" }}
+                    >
+                      Add new PII field
+                    </Text>
+                    <ManualButtonGroup
+                      isSubmitting={isSubmitting}
```

Comment:
Should the "Save" on this screen take them somewhere instead of leaving them here?

Reply:
Thanks for the usability improvements here, directing them back to the connectors screen!